lib/box/index.js

```diff
@@ -8,6 +8,7 @@ const fs = require('hexo-fs');
 const chalk = require('chalk');
 const { EventEmitter } = require('events');
 const micromatch = require('micromatch');
+const { ignore } = require('../hexo/default_config');
```

Comment:
```suggestion
const ignore = ['**/themes/*/node_modules/**', '**/themes/*/.git/**'];
```

`**/themes/*/node_modules/**` & `**/themes/*/.git/**` are supposed to be always ignored, right?

Since it is a simple array, I see no necessary to have an extra `require`.

Reply:
I put it in `default_config.js` is to centralize the config for maintainability, it would be easier to find it  if we need to add a new pattern in future.

Reply:
But it is not `default_config.js` designed to do. `default_config` is to provides default config if user doesn't specified.